modules/solid_properties/doc/moosedocs.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
```

Comment:
This should be a symlink to `../../combined/doc/moosedocs.py` instead (see another module for an example)